components/ConvertStepper/ConvertStepper.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import StepperLayout from './StepperLayout'
+import Step from './Step'
+import Divider from './Divider'
+
+const STEP_WIDTH = 180
+
+function ConvertStepper() {
+  return (
+    <StepperLayout
+      antCount="323424"
+      anjCount="54235"
```

Comment:
When it will be made dynamic, we could use `formatUnits()` here and get commas: https://github.com/aragon/convert.aragon.org/blob/master/lib/web3-utils.js#L111-L158